src/plugins/image/editing/image-editor.js

```diff
@@ -154,109 +197,149 @@ function ImageEditor({ content, onContentChanged, publicStorage, privateStorage
-                      value={effect.sourceUrl}
-                      onChange={handleEffectInternalSourceUrlValueChanged}
-                      />
-                    <StorageFilePicker
-                      publicStorage={publicStorage}
-                      privateStorage={privateStorage}
-                      fileName={effect.sourceUrl}
-                      onFileNameChanged={handleEffectInternalSourceUrlFileNameChanged}
-                      />
-                  </div>
+
+              {effect.type === EFFECT_TYPE.reveal && (
+              <Fragment>
+                <FormItem label={t('startPosition')} {...formItemLayout}>
+                  <InputNumber
+                    defaultValue={effect.startPosition}
+                    min={0}
+                    max={100}
+                    formatter={value => `${value}%`}
+                    parser={value => value.replace('%', '')}
+                    onChange={handleStartPositionValueChanged}
+                    />
                 </FormItem>
+                <FormItem label={t('orientationLabel')} {...formItemLayout}>
+                  <RadioGroup value={effect.orientation} onChange={handleOrientationValueChanged}>
+                    <RadioButton value={ORIENTATION.horizontal}>{t('orientationOptionHorizontal')}</RadioButton>
+                    <RadioButton value={ORIENTATION.vertical}>{t('orientationOptionVertical')}</RadioButton>
+                  </RadioGroup>
+                </FormItem>
+              </Fragment>
               )}
-              <Form.Item label={t('copyrightInfos')} {...formItemLayout}>
-                <TextArea value={effect.text} onChange={handleEffectCopyrightInfoValueChanged} autoSize={{ minRows: 3 }} />
-              </Form.Item>
-              {effect.type === EFFECT_TYPE.reveal && (
-                <div>
-                  <FormItem label={t('startPosition')} {...formItemLayout}>
-                    <InputNumber
-                      defaultValue={effect.startPosition}
-                      min={0}
-                      max={100}
-                      formatter={value => `${value}%`}
-                      parser={value => value.replace('%', '')}
-                      onChange={handleStartPositionValueChanged}
-                      />
-                  </FormItem>
-                  <FormItem label={t('orientationLabel')} {...formItemLayout}>
-                    <RadioGroup value={effect.orientation} onChange={handleOrientationValueChanged}>
-                      <RadioButton value={ORIENTATION.horizontal}>{t('orientationOptionHorizontal')}</RadioButton>
-                      <RadioButton value={ORIENTATION.vertical}>{t('orientationOptionVertical')}</RadioButton>
-                    </RadioGroup>
-                  </FormItem>
+
+              {effect.type === EFFECT_TYPE.clip && (
+                <div className="Image-clipEffect">
+                  {!!currentImageSource && (
```

Comment:
Just a small idea: It would be nice to have a line of text above the clip editor saying something like "Please use the selection rectangle to specify the displayed area" or similar.

Reply:
Good idea, I shall add that.
"Bitte der rechteckige Zuschnitt benutzen, um eine Bereich auszuwählen"?

Reply:
"Bitte benutzen Sie das Markierungsrechteck, um einen Bildbereich auszuwählen" comes to my mind

Reply:
Would have not come to my mind 😅 Thanks, done